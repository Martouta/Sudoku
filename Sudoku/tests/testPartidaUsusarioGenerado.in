2

2

1
2 0 1

1
3 1 4

1
2 2 4

1
1 2 3

1
2 3 3

1
3 3 1



Explicación:

Escribe las medidas n del tablero
2 //se trata de un sudoku 4x4

Selecciona la dificultad: facil(1), medio(2) y dificil(3)
2

1 //en este juego de pruebas sólo introducimos valores en las celdas para comprobar que es correcto
Introduce X, Y y valor
2 0 1

1
Introduce X, Y y valor
3 1 4

1
Introduce X, Y y valor
2 2 4

1
Introduce X, Y y valor
1 2 3

1
Introduce X, Y y valor
2 3 3

1
Introduce X, Y y valor
3 3 1
