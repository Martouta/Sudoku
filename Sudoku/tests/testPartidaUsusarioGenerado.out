Se ha insertado el valor 2 en la fila 3 y columna 2
Se ha ejecutado el algoritmo, 22 iteraciones
Se ha insertado el valor 1 en la fila 0 y columna 2
Se ha ejecutado el algoritmo, 31 iteraciones
Se ha insertado el valor 3 en la fila 0 y columna 1
Se ha ejecutado el algoritmo, 30 iteraciones
Se ha insertado el valor 2 en la fila 0 y columna 0
Se ha insertado el valor 3 en la fila 0 y columna 1
Se ha insertado el valor 1 en la fila 0 y columna 2
Se ha insertado el valor 4 en la fila 0 y columna 3
Se ha insertado el valor 4 en la fila 1 y columna 0
Se ha insertado el valor 1 en la fila 1 y columna 1
Se ha insertado el valor 3 en la fila 1 y columna 2
Se ha insertado el valor 2 en la fila 1 y columna 3
Se ha insertado el valor 1 en la fila 2 y columna 0
Se ha insertado el valor 2 en la fila 2 y columna 1
Se ha insertado el valor 4 en la fila 2 y columna 2
Se ha insertado el valor 3 en la fila 2 y columna 3
Se ha insertado el valor 3 en la fila 3 y columna 0
Se ha insertado el valor 4 en la fila 3 y columna 1
Se ha insertado el valor 2 en la fila 3 y columna 2
Se ha insertado el valor 1 en la fila 3 y columna 3
Se ha ejecutado el algoritmo, 30 iteraciones
Se ha ejecutado el algoritmo, 30 iteraciones
Se ha insertado el valor 4 en la fila 0 y columna 3
Se ha ejecutado el algoritmo, 18 iteraciones
Se ha insertado el valor 1 en la fila 1 y columna 1
Se ha ejecutado el algoritmo, 9 iteraciones
Se ha insertado el valor 2 en la fila 0 y columna 0
Se ha ejecutado el algoritmo, 9 iteraciones
Se ha insertado el valor 2 en la fila 1 y columna 3
Se ha ejecutado el algoritmo, 9 iteraciones
Se ha insertado el valor 2 en la fila 2 y columna 1
Se ha ejecutado el algoritmo, 9 iteraciones
Se ha insertado el valor 4 en la fila 1 y columna 0
Se ha ejecutado el algoritmo, 9 iteraciones
Se ha insertado el valor 3 en la fila 3 y columna 0
Se ha ejecutado el algoritmo, 1 iteraciones
Se ha insertado el valor 2 en la fila 0 y columna 0
Se ha insertado el valor 3 en la fila 0 y columna 1
Se ha insertado el valor 1 en la fila 0 y columna 2
Se ha insertado el valor 4 en la fila 0 y columna 3
Se ha insertado el valor 4 en la fila 1 y columna 0
Se ha insertado el valor 1 en la fila 1 y columna 1
Se ha insertado el valor 3 en la fila 1 y columna 2
Se ha insertado el valor 2 en la fila 1 y columna 3
Se ha insertado el valor 1 en la fila 2 y columna 0
Se ha insertado el valor 2 en la fila 2 y columna 1
Se ha insertado el valor 4 en la fila 2 y columna 2
Se ha insertado el valor 3 en la fila 2 y columna 3
Se ha insertado el valor 3 en la fila 3 y columna 0
Se ha insertado el valor 4 en la fila 3 y columna 1
Se ha insertado el valor 2 en la fila 3 y columna 2
Se ha insertado el valor 1 en la fila 3 y columna 3
Se ha ejecutado el algoritmo, 1 iteraciones
++-----++-----++
|| 2| 3|| 1| 4||
|| 4| 1|| 0| 2||
++-----++-----++
|| 0| 2|| 0| 0||
|| 3| 0|| 2| 0||
++-----++-----++


Se ha insertado el valor 1 en la fila 2 y columna 0
++-----++-----++
|| 2| 3|| 1| 4||
|| 4| 1|| 0| 2||
++-----++-----++
|| 1| 2|| 0| 0||
|| 3| 0|| 2| 0||
++-----++-----++


Se ha insertado el valor 4 en la fila 3 y columna 1
++-----++-----++
|| 2| 3|| 1| 4||
|| 4| 1|| 0| 2||
++-----++-----++
|| 1| 2|| 0| 0||
|| 3| 4|| 2| 0||
++-----++-----++


Se ha insertado el valor 4 en la fila 2 y columna 2
++-----++-----++
|| 2| 3|| 1| 4||
|| 4| 1|| 0| 2||
++-----++-----++
|| 1| 2|| 4| 0||
|| 3| 4|| 2| 0||
++-----++-----++


Se ha insertado el valor 3 en la fila 1 y columna 2
++-----++-----++
|| 2| 3|| 1| 4||
|| 4| 1|| 3| 2||
++-----++-----++
|| 1| 2|| 4| 0||
|| 3| 4|| 2| 0||
++-----++-----++


Se ha insertado el valor 3 en la fila 2 y columna 3
++-----++-----++
|| 2| 3|| 1| 4||
|| 4| 1|| 3| 2||
++-----++-----++
|| 1| 2|| 4| 3||
|| 3| 4|| 2| 0||
++-----++-----++


Se ha insertado el valor 1 en la fila 3 y columna 3
Has acabado el juego en 0 horas, 1 minutos i 12 segundos
El juego ha sido iniciado el Sun Nov 15 20:22:17 CET 2015y acabado el Sun Nov 15 20:23:29 CET 2015